libs/SalesforceSDKCore/SalesforceSDKCore/Classes/NativeLoginManager.swift

```diff
@@ -71,4 +71,64 @@ public protocol NativeLoginManager {
     ///
     /// Note: this call will dismiss your login view controller.
     @objc func biometricAuthenticationSuccess()
+    
+    // MARK: Headless, Password-Less Login Via One-Time-Passcode
+    
+    /// Submits a request for a one-time-passcode to the Salesforce headless password-less login flow.
+    /// This fulfills step three of the headless password-less login flow.
+    ///
+    /// See https://help.salesforce.com/s/articleView?id=sf.remoteaccess_headless_passwordless_login_public_clients.htm&type=5
+    ///
+    /// - Parameters:
+    ///   - username: A valid Salesforce username.  Note that email may be used for community users
+    ///   - recaptchaToken: A reCAPTCHA token provided by the reCAPTCHA SDK
+    ///   - otpVerificationMethod: The delivery method for the OTP
+    /// - Returns: An OTP request result with the overall login result and the OTP identifier for
+    /// successful OTP requests
+    ///
+    @objc func submitOtpRequest(
+        username: String,
+        reCaptchaToken: String,
+        otpVerificationMethod: OtpVerificationMethod) async throws -> OtpRequestResult
+    
+    /// Submits a request for a one-time-passcode to the Salesforce headless password-less login flow.
+    /// This fulfills steps eight, eleven and thirteen of the headless password-less login flow.
+    ///
+    /// See https://help.salesforce.com/s/articleView?id=sf.remoteaccess_headless_passwordless_login_public_clients.htm&type=5
+    ///
+    /// - Parameters:
+    ///   - otp: A user-entered OTP
+    ///   - otpIdentifier: The OTP identifier issued by the Headless Identity API
+    ///   - otpVerificationMethod: The OTP verification method used to obtain the OTP identifier
+    /// - Returns: A login result indicating the outcome of the authorization and access token requests
+    ///
+    @objc func submitPasswordlessAuthorizationRequest(
+        otp: String,
+        otpIdentifier: String,
+        otpVerificationMethod: OtpVerificationMethod
+    ) async throws -> NativeLoginResult
+}
+
+/// An Objective-C compatible OTP request result
+@objc public class OtpRequestResult: NSObject {
+    
+    /// The overall result of the OTP request.
+    public let nativeLoginResult: NativeLoginResult
```

Comment:
should `nativeLoginResult` and `otpIdentifier` be exposed to objc too?

Reply:
Is there a way to cascade the declaration from the class?

Reply:
I haven't tried it before but there's this https://www.hackingwithswift.com/example-code/language/what-is-the-objcmembers-attribute

Reply:
That's the one I was thinking of.  I have used that quite a bit and this type likely won't have members that are Swift-only.